# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/hotfix-sparc64-qemu-initrd-page-size.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

Initramfs unpacking failed: invalid magic at start of compressed archive
No filesystem could mount root, tried:

Kernel panic - not syncing: VFS: Unable to mount root fs on unknown-block(0,0)

--- linux-6.6/arch/sparc/Makefile.vanilla	2023-12-18 12:27:25.366850564 +0100
+++ linux-6.6/arch/sparc/Makefile	2023-12-18 12:29:14.298845914 +0100
@@ -40,7 +40,7 @@
 #
 
 CHECKFLAGS    += -D__sparc__ -D__sparc_v9__ -D__arch64__
-KBUILD_LDFLAGS := -m elf64_sparc
+KBUILD_LDFLAGS := -m elf64_sparc -z max-page-size=0x100000
 export BITS   := 64
 UTS_MACHINE   := sparc64
 
